osde2e/tests/aws_vpce_operator_tests.go

```diff
@@ -0,0 +1,8 @@
+package osde2e_tests
```

Comment:
Package names cannot have underscores.

This should be renamed to "tests" or "osde2etests".

Reply:
It doesn't look like this PR has the latest boilerplate? https://github.com/openshift/boilerplate/blob/master/boilerplate/openshift/golang-osd-operator-osde2e/e2e-harness-generate.sh#L45